py/mpprint.h

```diff
@@ -80,3 +80,52 @@ int mp_vprintf(const mp_print_t *print, const char *fmt, va_list args);
 #endif
 
 #endif // MICROPY_INCLUDED_PY_MPPRINT_H
```

Comment:
This line must be at the end of the file.

Reply:
And yet no.
We need to redefine MP_DEBUG_PRINT in each individual file.